src/app/snapshot/snapshots-page/snapshot-filter/snapshot-filter.component.ts

```diff
@@ -55,6 +57,8 @@ export class SnapshotFilterComponent implements OnInit {
   public queryChange = new EventEmitter();
   @Output()
   public viewModeChange = new EventEmitter<SnapshotPageViewMode>();
+  @Output()
```

Comment:
I think we'd better refactor this component, so that VM view logic is separated from Volume view logic